components/textfield/stories/template.js

```diff
@@ -28,6 +29,9 @@ export const Template = ({
 	isReadOnly = false,
 	isKeyboardFocused = false,
 	isLoading = false,
+	displayLabel = false,
+	sideLabel = false,
```

Comment:
Minor but I think the naming of this variable - sideLabel - made me think it was separate content. For variables that are boolean, I'd like to see a naming that implies that such as `isSideLabel`.